core/libs/commonwealth/commonwealth/utils/general.py

```diff
@@ -1,8 +1,15 @@
 import os
+import subprocess
+from pathlib import Path
 
 from loguru import logger
 
 
+def file_is_open(path: Path) -> bool:
+    result = subprocess.run(['lsof', path.resolve()], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
```suggestion
    result = subprocess.run(["lsof", path.resolve()], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

just to make the linter happy